### DISCUSSION TITLE
Laragear packages

### DISCUSSION THREAD

**Discussion by DarkGhostHunter** [HassanZahirnia/laravel-package-ocean#145] [Package Suggestions]
Hi!

I'm the author and maintainer of several Laravel packages under the Laragear namespace. Most notably:

- [WebAuthn](https://github.com/Laragear/WebAuthn): Passkeys authentication system
- [TwoFactor](https://github.com/Laragear/TwoFactor): 2FA authentication system
- [Meta](https://github.com/Laragear/Meta): A package to help common tasks of a package.
- [MetaTesting](https://github.com/Laragear/MetaTesting): A package to test your package.
- [Preload](https://github.com/Laragear/Preload): A auto-generated PHP 7.4+ preloader
- [CacheQuery](https://github.com/Laragear/CacheQuery): Cache database queries, even nested Eloquent calls.
- [RUT](https://github.com/Laragear/Rut): Tools to handle chilean RUTs (citizen ID).
- [ReCaptcha](https://github.com/Laragear/ReCaptcha): ReCaptcha handler that uses HTTP/2.
- [Poke](https://github.com/Laragear/Poke): Avoids `TokenMismatchException` by keeping the token alive.
- [Json](https://github.com/Laragear/Json): Helper to create and manipulate JSON.
- [ApiManager](https://github.com/Laragear/ApiManager): Helper to manage multiple REST servers.
- [Alerts](https://github.com/Laragear/Alerts): System to create multiple alerts on the frontend.
- [Compare](https://github.com/Laragear/Compare): Helper to compare values fluently with human words.

I would be cool to have at last some of these listed to help developers.

**Comment by HassanZahirnia** [comment]
Thanks for submitting all your packages 😊 I've added most of them.

The only problem I noticed was in the README of "Compare" package where the composer package name is wrong.

**Reply by DarkGhostHunter** [reply]
Thanks for pointing that out. Guess I copyasted that wrong.
